### ISSUE TITLE
Is Boot2Docker deprecated?

### ISSUE THREAD

**Issue by rcross** [boot2docker/boot2docker#1065]
I'm a bit confused. The latest version of Docker seems to deprecate boot2docker, but I also see this is a separate project and is apparently still being worked on?

Can anyone give me some insight into what's going on?

**Comment by thaJeztah** [comment]
I can imagine this is confusing; boot2docker exists of *two* parts; boot2docker-cli and boot2docker - a lightweight Linux distribution for running Docker.

boot2docker-cli is a tool to setup a Virtual Machine running the boot2docker distro. This tool (boot2docker-cli) has been deprecated in favor of docker-machine (which also lets you setup a Virtual Machine running boot2docker Linux)

So TL;DR, boot2docker (the Linux distro) is **not** deprecated (but there *will* be a large change in an upcoming release)

**Comment by jakirkham** [comment]
@thaJeztah, by large change in an upcoming release are you referring to the fact the VM will be Debian-based instead of Tiny Core-based or are you meaning something different?

**Comment by thaJeztah** [comment]
@jakirkham yes, that's the change I was referring to

**Comment by md5** [comment]
@thaJeztah Is [`unclejack/debian2docker`](https://github.com/unclejack/debian2docker) the repo we should be looking at or is there another proposed ISO out there?

**Comment by thaJeztah** [comment]
No, that project is unrelated (apart from @unclejack being a Docker maintainer), the next generation of the boot2docker distribution can be found here; https://github.com/docker/boot2docker

**Comment by md5** [comment]
:+1:
